source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,197 @@ Install with Confluent Cloud
+      Otherwise, the Essentials tier is enough. 
+
+   .. step:: Create a cluster
+
+      Navigate to your :guilabel:`Environment` page to click :guilabel:`Add Cluster`. 
+
+      For most use cases, the Basic and Standard tier clusters are enough. If you 
+      are migrating large volumes of data, consider the Dedicated cluster tier.
+
+   .. step:: Configure permissions and gather API keys
+
+      To configure Relational Migrator, you must configure and gather the following 
+      permissions and API keys to include them in your ``user.properties`` file.
+      The location of your ``user.properties`` file varies according to your operating
+      system. For more information, see :ref:`file-location`. 
+
+      The following API keys are found in the Confluent Cloud console. For more 
+      information see, `Use API Keys <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+      .. procedure::
+
+         .. step:: Configure Confluent Cloud API keys
+
+            Click the :icon-fa5:`bars` icon on the top right corner, then under 
+            :guilabel:`Administration`, select :guilabel:`Cloud API Keys`.
+
+            Create Confluent Cloud API keys and configure permissions for them. You 
+            have two permission options:
+
+            1. Grant global access.
+         
+            2. Create a service account and grant granular access.
+
+               If you create a service account, you must add the ``Operator`` role for the 
+               environment and the ``CloudClusterAdmin`` role to the cluster. 
+
+            Once created, save the API key and secret. These values will be added to your 
+            ``user.properties`` file as ``migrator.confluent.cloud-credentials.api-key`` 
+            and ``migrator.confluent.cloud-credentials.api-secret``.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` and on the left-side navigation 
+            click :guilabel:`API Keys` to create your Kafka cluster API keys.
+
+            Once created, save the API key and secret. These values will be added to your 
+            ``user.properties`` file as ``migrator.confluent.kafka-credentials.api-key`` 
+            and ``migrator.confluent.kafka-credentials.api-secret``.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather more configuration information
+
+      The following information is found in the Confluent Cloud console. For more 
```

Comment:
As above, let's avoid "the following." Also, let's avoid passive voice. Consider, 

"You'll find your environment ID, cluster ID, Bootstrap server, and cluster REST endpoint in the Confluent Cloud console."

Reply:
Updated.